openedx_tagging/core/tagging/models/base.py

```diff
@@ -94,7 +94,9 @@ def __str__(self):
         """
         User-facing string representation of a Tag.
         """
-        return f"<{self.__class__.__name__}> ({self.id}) {self.value}"
+        if self.external_id:
+            return f"<{self.__class__.__name__}> ({self.id} / {self.external_id} / {self.value})"
+        return f"<{self.__class__.__name__}> ({self.id} / {self.value})"
```

Comment:
The `id` is not necessarily going to be stable, so I think this may make the tests flaky. And it doesn't mean anything to users. What do you think about just using `value` and `external_id` in these __str__ and leaving out `id` ?

Reply:
The problem is that this function is also used by developers when printing tags in the console. I originally had the id, I wouldn't be too sure if it's a good idea to remove the id, What do you think?

Reply:
I don't think we should display the ID to users. You could add a separate function like `.display_str()` that doesn't have the ID and use that instead of `__str__()` if you want to keep the ID in `__str__`.

Reply:
Thanks, Done. Udpated here https://github.com/openedx/openedx-learning/pull/135/commits/9030b1e8d56fddb7d997dd4b14c4fec4975528d9